repository_test.go

```diff
@@ -73,3 +73,46 @@ func TestRepositoryIsEmpty(t *testing.T) {
 		t.Error("Repository should not be empty")
 	}
 }
+
+func TestRepositoryChangePassword(t *testing.T) {
+	testPassword := "this_is_a_password"
+	newPassowrd := "this_is_another_password"
+
+	dir, err := ioutil.TempDir("", "knoxite")
+	if err != nil {
+		t.Errorf("Failed creating temporary dir for repository: %s", err)
+		return
+	}
+	defer os.RemoveAll(dir)
+
+	_, err = NewRepository(dir, testPassword)
+	if err != nil {
+		t.Errorf("Failed creating repository: %s", err)
+		return
+	}
+
+	repo, err := OpenRepository(dir, testPassword)
+	if err != nil {
+		t.Errorf("Failed opening repository: %s", err)
+		return
+	}
+
+	err = repo.ChangePassword(newPassowrd)
```

Comment:
Typo in variable name: `newPassword`

---

You could also simplify this a little bit:
```go
if !repo.ChangePassword(newPassword) {
   // ...
}
```